src/models/PMCMean.h

```diff
@@ -0,0 +1,37 @@
+#ifndef PMCMEAN
+#define PMCMEAN
+#include <stdint.h>
```

Comment:
`stdint.h` er deprecated i C++, brug `cstdint` i stedet for.

Reply:
fixed